pages/advanced/erc-7579/7579-safe.mdx

```diff
@@ -0,0 +1,140 @@
+import { Callout, Steps } from 'nextra/components'
+import CustomCard from '../../../components/CustomCard'
+
+# Safe and ERC-7579
+
+<Callout emoji="⚠️">
+  The Safe7579 Adapter is under audit, with an estimated completion date of June 14, 2024.
+</Callout>
+
+The Safe7579 Adapter is a smart contract developed by Rhinestone and Safe to make Safe Smart Accounts compliant with ERC-7579. Through this, [14 audited modules](https://github.com/rhinestonewtf/modulekit/tree/examples/audit-ready/examples/src) developed by Rhinestone will be available for builders building with the Safe7579 Adapter, such as a dead man switch, flash-loan, social recovery, etc.
+
+![diagram-safe-7579](../../../assets/diagram-safe-7579.png)
+
```

Comment:
Might make sense to:
1. List the "official 7579" website (in the hopes that more and more stuff gets added there
2. Add a bit more fluff as to why it is cool (one ecosystem across account, so builders target all major wallets; registry provides per-transaction security checks on modules, so modules with security compromises are automatically disabled for your account; etc.)